qemu/tests/numa_stress.py

```diff
@@ -68,57 +71,109 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment.
     """
+
+    def heavyload_install():
+        test_installed_cmd = 'dir "%s" | findstr /I heavyload' % install_path
+        if session.cmd_status(test_installed_cmd) != 0:
+            logging.warning("Could not find installed heavyload in guest, will"
+                            " install it via winutils.iso ")
+            winutil_drive = utils_misc.get_winutils_vol(session)
+            if not winutil_drive:
+                test.cancel("WIN_UTILS CDROM not found.")
+            install_cmd = params["install_cmd"] % winutil_drive
+            session.cmd(install_cmd)
+
+    def run_stress():
+        if os_type == 'linux':
+            start_cmd = "stress %s > /dev/null &" % stress_args
+            session.cmd(start_cmd)
```

Comment:
When review `Stress`, i got one comment in `utils_test.Stress`
```
# The background process sometimes does not return to                   
# terminate, if timeout, send a blank line afterward 
```
Then safe mode is better to avoid this issue.

Reply:
Thanks, updated.